src/chainparams.cpp

```diff
@@ -760,7 +760,7 @@ class CRegTestParams : public CChainParams {
     explicit CRegTestParams() {
         strNetworkID = "regtest";
         bool isJellyfish = false;
-        isJellyfish = gArgs.GetBoolArg("-jellyfish_regtest", false);
+        isJellyfish = gArgs.GetBoolArg("-regtest-skip-loan-collateral-validation", false);
```

Comment:
I'd rather we keep `-jellyfish_regtest` and add an additional flag for skipping `LoanCollateralFactor` validation.
It's confusing now seeing as `isJellyfish` has lots of side effect already. See https://github.com/DeFiCh/ain/blob/4b3c187f590738f44f5574901f1fec233a53c226/src/chainparams.cpp#L884

Reply:
Agree with Jeremy. Do not touch `jellyfish_regtest`. I'd leave it as it is. The `regtest-skip-loan-collateral-validation` is an additional arg, that is added in addition, not intended to replace jellyfish_regtest.